app/models/trip.rb

```diff
@@ -0,0 +1,18 @@
+class Trip < ApplicationRecord
+  validates :rating, inclusion: { in: [1, 2, 3, 4, 5], allow_nil: true}
+  belongs_to :driver
+  belongs_to :passenger
+  
+  def converter
+    cost = @trip.cost
+    @converted_cost = converter(cost)
+  end
+  
+  def converter(integer)
+    integer = integer.to_s
+    cents = integer[-2..-1]
+    dollars = integer[0..-3]
+    return_statement = "#{dollars}.#{cents}"
+  end
```

Comment:
Love that y'all made a method for this! That said, the fact that this is repeated in three different classes is a sign that it probably belongs in its own class. It could be put in a class (maybe a new class called Money, for instance) and made a class method so that an instance of money isn't needed to call it.